### DISCUSSION TITLE
Vite + React app shows no eslint warnings and errors.

### DISCUSSION THREAD

**Discussion by zwelhtetyan** [vitejs/vite-plugin-react#49] [Q&A] [answered]
How to config eslint in vite-react app?

**Comment by ArnaudBarre** [comment, marked as answer]
Vite does not come with eslint support by default. 

I wrote this plugin for this: https://www.npmjs.com/package/@nabla/vite-plugin-eslint

**Comment by Karim8090** [comment]
> How to config eslint in vite-react app?

#1

**Comment by Karim8090** [comment]
> Vite does not come with eslint support by default. 
> 
> I wrote this plugin for this: https://www.npmjs.com/package/@nabla/vite-plugin-eslint

#1

**Reply by Karim8090** [reply]
Babuuuu 1996

**Reply by Karim8090** [reply]
#1

**Comment by Karim8090** [comment]
Babuuuu 1996

**Reply by Karim8090** [reply]
#1

**Comment by Karim8090** [comment]
> How to config eslint in vite-react app?

#49

**Reply by Karim8090** [reply]
#49

**Reply by Karim8090** [reply]
Approved
